mapbox/services/uploads.py

```diff
@@ -7,6 +9,16 @@
 from mapbox.services.base import Service
 
 
+@contextmanager
+def CacheBuster(session):
+    """Wraps a session with a randomized query parameter `_`
+    to discourage cached results.
+    """
+    session.params.update(_=str(uuid.uuid1()))
+    yield
+    session.params.pop('_')
```

Comment:
The contextlib yield thing is weird but cool! `_` is a legit query string key and seems to indicate that it's not meaningful. @perrygeo I suppose there's a small risk that our API server could choke on a unexpected query. Other than that, everything looks good to me.